scripts/fetch_meca_archives.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+# This requires jq
+# pass all or new as first param, the output directory as second, manuscripts text file as third, and meca lookup file as fourth
+# example run:
+# ./scripts/fetch_meca_archives.sh [all_or_new] [output_dir] [manuscripts_txt] [meca_lookup]
+
+SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+PARENT_DIR="$(dirname "${SCRIPT_DIR}")"
+
+all_or_new="${1-new}" # default new
+
+if [[ "$all_or_new" != "all" && "$all_or_new" != "new" ]]; then
+    echo "Error: all_or_new is not 'all' or 'new'"
+    exit 1
+fi
+
+output_dir="$(realpath ${2-${PARENT_DIR}/incoming})" # default ./incoming
+manuscripts_txt="$(realpath ${3-${PARENT_DIR}/manuscripts.txt})" # default ./manuscripts.txt
+meca_lookup="$(realpath ${3-${PARENT_DIR}/meca-lookup.txt})" # default ./meca-lookup.txt
+
+echo "Retrieving ${all_or_new} manuscripts in ${manuscripts_txt}"
+
+while read -r line; do
+    if [[ "$all_or_new" = "all" || ! -d "${PARENT_DIR}/data/${line}" ]]; then
+        $SCRIPT_DIR/fetch_meca_archive.sh "$line" $output_dir $meca_lookup
```

Comment:
Was trialling this over in https://github.com/elifesciences/enhanced-preprints-data/pull/54 with 10.21203/rs.3.rs-2200020/v3.

When I run `./scripts/fetch_meca_archives.sh`, looks like I get an error when trying to invoke `fetch_meca_archive.sh`. Looks like it's due to `readlink`s `-f` option and not being able to retrieve `$SCRIPT_DIR`. Is there a dependency (aside from `jq`) I need to install?

```
$ ./scripts/fetch_meca_archives.sh     
                                             
readlink: illegal option -- f
usage: readlink [-n] [file ...]
Retrieving new manuscripts in /Users/fredatherden/Documents/GitHub/enhanced-preprints-data/manuscripts.txt
./scripts/fetch_meca_archives.sh: line 27: ./fetch_meca_archive.sh: No such file or directory
```

Reply:
Nevermind. Aliasing readlink to greadlink worked for me

Reply:
`jq`, `sed` and `xmllint` are needed. I could run through docker if you anticipate any issues.

Reply:
Working fine for me locally after the workaround above.